src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -490,4 +490,10 @@ Blank()
 "one"
 
 >> Boolean(IfError(ParseJSON("{""field1"":true}").field1, ParseJSON("false")))
-true
\ No newline at end of file
+true
+
+>> Text(ParseJSON("1.234"), "#.##")
+"1.23"
+
+>> Text(ParseJSON("1.234"), "#.##", "fr-fr")
+"1,23"
```

Comment:
Add some tests with blank values and 2+ arguments:

    >> Text(ParseJSON("null"), "0.00")
    ""

    >> Text(ParseJSON("null"), "0.00", "de-DE")
    ""

Reply:
Done